certifi/core.py

```diff
@@ -18,10 +20,24 @@ class DeprecatedBundleWarning(DeprecationWarning):
     """
 
 
-def where():
-    f = os.path.dirname(__file__)
+_cert_tempfile = None
+_cert_path = os.path.join(os.path.dirname(__file__), 'cacert.pem')
```

Comment:
This line will fail in the environments that don't have `__file__` defined, such as cx_Freeze or PyOxidizer.

The fix is IMHO quite simple - to catch `NameError` from this line. The rest should work fine - `pkgutil.get_data` works cross platform. 

It would be great to get this into a release. At the moment, it's quite hard to package a single executable depending on e. g. `requests` library.